packages/client/src/views/SearchResult/AdvancedSearch.tsx

```diff
@@ -85,16 +76,33 @@ export const isAdvancedSearchFormValid = (value: IAdvancedSearchFormState) => {
       Boolean(value[key as keyof IAdvancedSearchFormState])
   )
   //handle date fields separately
-  const validDateFields = dateFieldTypes.filter(
-    (key) =>
-      value[key as keyof IAdvancedSearchFormState] &&
-      isValidDateRangePickerValue(
-        value[key as keyof IAdvancedSearchFormState] as IDateRangePickerValue
-      )
+  const validationResultsPerDateField: boolean[] = []
+
+  for (const key of dateFieldTypes) {
+    const dateObj = value[
+      key as keyof IAdvancedSearchFormState
+    ] as IDateRangePickerValue
+
+    if (!dateObj) continue
+
+    if (isValidDateRangePickerValue(dateObj)) {
+      validationResultsPerDateField.push(true)
+      continue
+    }
+    /*
+     * if isValidDateRangePickerValue couldn't validate dateObj
+     * the date field would have an invalid value
+     */
+    if (dateObj.exact) validationResultsPerDateField.push(false)
+  }
+
+  const validDateFields = validationResultsPerDateField.filter((valid) => valid)
+  const invalidDateFields = validationResultsPerDateField.filter(
+    (valid) => !valid
```

Comment:
```suggestion
```

Reply:
`validDateFields` is used to determine the validCount and it may look cleaner than to `filter` them in another constant